cmd/cert-checker/main.go

```diff
@@ -141,9 +143,9 @@ func (c *certChecker) getCerts(unexpiredOnly bool) error {
 	return nil
 }
 
-func (c *certChecker) processCerts(wg *sync.WaitGroup, badResultsOnly bool) {
+func (c *certChecker) processCerts(ctx context.Context, wg *sync.WaitGroup, badResultsOnly bool) {
```

Comment:
I don't think `Context`s are needed here, we don't cross any RPC boundaries and they aren't used anywhere.